dev/breeze/src/airflow_breeze/utils/selective_checks.py

```diff
@@ -776,7 +778,21 @@ def runs_on(self) -> str:
         if self._github_repository == APACHE_AIRFLOW_GITHUB_REPOSITORY:
             if self._github_event in [GithubEvents.SCHEDULE, GithubEvents.PUSH]:
                 return RUNS_ON_SELF_HOSTED_RUNNER
-            if self._github_actor in COMMITTERS and USE_PUBLIC_RUNNERS_LABEL not in self._pr_labels:
+            actor = self._github_actor
+            if self._github_event in (GithubEvents.PULL_REQUEST, GithubEvents.PULL_REQUEST_TARGET):
+                try:
+                    actor = self._github_context_dict["event"]["pull_request"]["user"]["login"]
+                    get_console().print(
+                        f"[warning]The {actor} retrieved as actor from GITHUB_CONTEXT's"
```

Comment:
```suggestion
                        f"[info]{actor} retrieved as actor from GITHUB_CONTEXT's"
```

2 things:

- Don't need "the" here, even if "The potiuk retrieved as actor" does sounds good :)
- Shouldn't this be an info, not warning?

Reply:
Warning is better. The colors to show are specifically designed to be easily distinguishable.

![image](https://github.com/apache/airflow/assets/595491/18573d00-95eb-4fab-a403-c7057dc79f1f)


* Don't need "the" here, even if "The potiuk retrieved as actor" does sounds good :)

I wanted to start the sentence capitalized.

Reply:
I changed it to:

```
                        f"[warning]The actor: {actor} retrieved from GITHUB_CONTEXT's"
```